index.js

```diff
@@ -51,8 +51,15 @@ var typeRegExp = /^[!#$%&'\*\+\-\.\^_`\|~0-9A-Za-z]+\/[!#$%&'\*\+\-\.\^_`\|~0-9A
  * @public
  */
 
-exports.format = format
-exports.parse = parse
+if(typeof exports !== 'undefined') {
```

Comment:
Do not add this code fork to what is published to npm. Also, this looks like it makes it impossible to add a dependency, since we would need require(). Support for doing that is required to accept this PR.